src/backend/base/langflow/components/models/AIMLModel.py

```diff
@@ -1,148 +1,128 @@
+        DictInput(
+            name="output_schema",
+            is_list=True,
+            display_name="Schema",
             advanced=True,
+            info="The schema for the Output of the model. You must pass the word JSON in the prompt. If left blank, JSON mode will be disabled.",
         ),
-        FloatInput(
-            name="top_p",
-            display_name="Top P",
-            info="Works together with top-k. (Default: 0.9)",
-            advanced=True,
+        DropdownInput(
+            name="model_name",
+            display_name="Model Name",
+            advanced=False,
+            options=AIML_CHAT_MODELS,
+            value=AIML_CHAT_MODELS[0],
         ),
-        FloatInput(
-            name="repeat_penalty",
-            display_name="Repeat Penalty",
-            info="Penalty for repetitions in generated text. (Default: 1.1)",
+        StrInput(
+            name="aiml_api_base",
+            display_name="AIML API Base",
             advanced=True,
+            info="The base URL of the OpenAI API. Defaults to https://api.aimlapi.com . You can change this to use other APIs like JinaChat, LocalAI e Prem.",
         ),
-        FloatInput(
-            name="temperature",
-            display_name="Temperature",
-            value=0.2,
-            info="Controls the creativity of model responses.",
+        SecretStrInput(
+            name="api_key",
+            display_name="AIML API Key",
+            info="The AIML API Key to use for the OpenAI model.",
+            advanced=False,
+            value="AIML_API_KEY",
         ),
-        StrInput(
-            name="system_message",
-            display_name="System Message",
-            info="System message to pass to the model.",
+        FloatInput(name="temperature", display_name="Temperature", value=0.1),
+        IntInput(
+            name="seed",
+            display_name="Seed",
+            info="The seed controls the reproducibility of the job.",
             advanced=True,
+            value=1,
         ),
     ]
 
-    def make_request(self) -> Message:
-        api_key = SecretStr(self.aiml_api_key) if self.aiml_api_key else None
-
-        headers = {
-            "Content-Type": "application/json",
-            "Authorization": f"Bearer {api_key.get_secret_value()}" if api_key else "",
-        }
-
-        messages = []
-        if self.system_message:
-            messages.append({"role": "system", "content": self.system_message})
+    def build_model(self) -> LanguageModel:  # type: ignore[type-var]
+        output_schema_dict: dict[str, str] = reduce(operator.ior, self.output_schema or {}, {})
+        aiml_api_key = self.api_key
+        temperature = self.temperature
+        model_name: str = self.model_name
+        max_tokens = self.max_tokens
+        model_kwargs = self.model_kwargs or {}
+        aiml_api_base = self.aiml_api_base or "https://api.aimlapi.com"
+        json_mode = bool(output_schema_dict) or self.json_mode
```

Comment:
Seems like `output_schema` exists only to determine whether to set `json_mode`. I do have the concern that even if fixed, the AI/ML API wouldn't support this (or, only would for specific models, which may be confusing for users).